src/main/drivers/dshot_command.h

```diff
@@ -73,4 +73,4 @@ bool dshotCommandQueueEmpty(void);
 bool dshotCommandIsProcessing(void);
 uint8_t dshotCommandGetCurrent(uint8_t index);
 bool dshotCommandOutputIsEnabled(uint8_t motorCount);
-bool dshotCommandsAreEnabled(dshotCommandType_e commandType);
+bool dshotStreamingCommandsAreEnabled();
```

Comment:
Is think you forgot `(void)` ?

Reply:
-s (line 76)

Reply:
line 75 puzzles me; still using this function in dshot_command.c , maybe it is declared in another header file?

Reply:
Fixed. Not sure what is puzzling about line 75?

Reply:
Sorry I mean removing line 76, it looks to me declaration of function `bool dshotCommandsAreEnabled(dshotCommandType_e commandType);` is still used?

Reply:
It's only used internally, so it's not part of the interface of the unit (i.e. the header file) any more.

Reply:
Changing the function declaration to static should already have told me this was for internal use only. Thanks for solving my little puzzle 🥇